kroxylicious-api/src/test/java/io/kroxylicious/proxy/filter/FilterInvokersTest.java

```diff
@@ -40,12 +41,8 @@ public void testValidFilters(KrpcFilter invalid) {
     public void testCompositeFilter() {
         MultipleSpecificFilter filterA = new MultipleSpecificFilter();
         RequestResponseFilter filterB = new RequestResponseFilter();
-        RequestFilter filterC = (apiKey, header, body, filterContext) -> {
-
-        };
-        ResponseFilter filterD = (apiKey, header, body, filterContext) -> {
-
-        };
+        RequestFilter filterC = (apiKey, header, body, filterContext) -> null;
```

Comment:
Hmm this implies we should have a null check in [`FilterHandler`](url)https://github.com/kroxylicious/kroxylicious/blob/1368705fc1d5fcc32960363d7c3cdd30ec56ad39/kroxylicious/src/main/java/io/kroxylicious/proxy/internal/FilterHandler.java#L65

Reply:
Added guard.   We need the API docs to make clear that filters must return a filer, and talk about behaviour if they throw an unchecked exception.